src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -273,21 +273,24 @@ public static void performMinimization(
    *     entirely
    */
   private static boolean isEmptyCompilationUnit(CompilationUnit cu) {
-    for (Node child : cu.getChildNodes()) {
-      if (child instanceof PackageDeclaration) {
-        // Package declarations don't count for the purposes of
-        // deciding whether to entirely remove a compilation unit.
-        continue;
-      } else if (child instanceof ClassOrInterfaceDeclaration) {
-        ClassOrInterfaceDeclaration cdecl =
-            ((ClassOrInterfaceDeclaration) child).asClassOrInterfaceDeclaration();
-        if (!cdecl.getMembers().isEmpty()) {
-          return false;
+      for (Node child : cu.getChildNodes()) {
+        if (child instanceof PackageDeclaration) {
+          // Package declarations don't count for the purposes of
+          // deciding whether to entirely remove a compilation unit.
+          continue;
         }
-      } else {
+
+        // If the node is a ClassOrInterfaceDeclaration, check if it has members.
+        if (child instanceof ClassOrInterfaceDeclaration cdecl) {
+          if (!cdecl.getMembers().isEmpty()) {
+            return false;
+          }
```

Comment:
If this falls through, this function will just return false below, which I don't think is desirable. I think we can simplify this further:
* if `cdecl.getMembers().isEmpty()` is true, continue
* otherwise, return false

Reply:
I completely overlooked this during the refactor! Thanks..

Reply:
Since we are using Java 11, I will remove the instanceof.